thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/dashboard/resources/AnomalyFlattenResource.java

```diff
@@ -0,0 +1,123 @@
+package org.apache.pinot.thirdeye.dashboard.resources;
```

Comment:
Can you include the apache license header. You can borough it from another class.

Reply:
Will update it. thanks